src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -952,4 +973,34 @@ protected Void call() {
       return task;
     }
   }
+
+  // ConvertToCdfService runs a CDF conversion in the background
+  private static class ConvertToCdfService extends Service<Void> {
+
+    private final String configPath;
+
+    ConvertToCdfService(String configPath) {
+      this.configPath = configPath;
+    }
+
+    @Override
+    protected Task<Void> createTask() {
+      Task<Void> task =
+          new Task<>() {
+            @Override
+            protected Void call() {
+              TabulatorSession session = new TabulatorSession(configPath);
+              session.convertToCdf();
+              return null;
```

Comment:
If the return type is Void, why do you have to have an explicit return statement?

Reply:
Pretty sure it was boilerplate for thread management. [This](https://stackoverflow.com/a/15425295/3846321) might shed some light on that.